src/features/content-answers/ContentAnswersModalContent.tsx

```diff
@@ -24,11 +25,13 @@ const ContentAnswersModalContent = ({ currentUser, fileName, isLoading, question
             <WelcomeMessage fileName={fileName} />
             <ul>
                 {questions &&
-                    questions.map(({ prompt, answer = '' }, index) => {
+                    questions.map(({ prompt, error, answer = '' }, index) => {
+                        const hasError = !!error;
                         return (
                             <li key={index}>
                                 <Question currentUser={currentUser} prompt={prompt} />
-                                <Answer answer={answer} isLoading={isLoading} />
+                                {!hasError && <Answer answer={answer} isLoading={isLoading} />}
+                                {hasError && !isLoading && <InlineError />}
```

Comment:
do we want to remove the `isLoading` check for `InlineError`? wouldn't this affect all "error answers"?

**Stage 1**
> Q: summarize this document

**Stage 2**
> Q: summarize this document
A: ERROR

**Stage 3**
> Q: summarize this document
A: ERROR
Q: summarize this document in 200 words

**Stage 4**
> Q: summarize this document
A: LOADING
Q: summarize this document in 200 words
A: LOADING

**Stage 5**
> Q: summarize this document
A: ERROR
Q: summarize this document in 200 words
A: blah blah blah

Reply:
```
{hasError ? <InlineError /> : <Answer answer={answer} isLoading={isLoading} />}
```